### ISSUE TITLE
Releasing to maven repository

### ISSUE THREAD

**Issue by computerlyrik** [nkolban/jpigpio#15]
Hi! For easier development i am trying to release the Build to Maven repository.

I will also update the Documentation about Build & Install some time in the future.

Ticket for requesting a repo at sonatype: https://issues.sonatype.org/browse/OSSRH-28498

I added @nkolban and @sovcik  to the developers in metadata of the parent pom.xml.

If i can do something else, please tell me!
Thank you very much for developing this project so far!

**Comment by computerlyrik** [comment]
PS: Due to currently working on master, i cannot guarantee a working base each second, but you may checkout https://github.com/computerlyrik/jpigpio/tree/8757603b34168f5fa677da7946eddb84d48be627 and type in `mvn clean package` after installing `maven` (and java jdk8).
All relevant Files should be outputted inside the `target/` Folder (.h .so and .jar).

I would be happy if you test this on your raspberry pi!

**Comment by nkolban** [comment]
I am no longer actively maintaining this project ... would you like to take ownership?

**Comment by sovcik** [comment]
@nkolban I can try keeping it alive :-) For my own selfish reasons :-) So, unless @computerlyrik is not willing to take ownership, then I can.

**Comment by nkolban** [comment]
Can I ask you @sovcik to chat with @computerlyrik and sort out between you who wants it?  From there we'll transfer ownership.

**Comment by computerlyrik** [comment]
@sovcik You took ownership? Thank you! :)

**Comment by nkolban** [comment]
I don't think so ... once ownership is decided ... the repository will move to under the new owners domain in Github.

**Comment by sovcik** [comment]
@computerlyrik, as I said, if you do not want to take over, then I can try maintaining it further. So, would you like to maintain or shall I? 
I'm no Java professional - only started with Java some year ago, so I might be not the most competent :-)
But the fact is, that I'm using this library in one of my live projects, so I've got real interest in keeping it alive.

**Comment by computerlyrik** [comment]
@sovcik I am still not sure, wether this lib makes it into one of my projects. Still in progress.
I think you are more destined to take over the project.

And i do not have the time to maintain this project accordingly. 
I feel glad if i can just present some pull requests to you for possible integrations :)

**Comment by sovcik** [comment]
@computerlyrik OK, understand.
@nkolban Neil, if it is fine for you, then I can take over. J

**Comment by nkolban** [comment]
@sovcik Perfect ... let us now study on what it takes to "move" a repository

**Comment by Xeli** [comment]
@computerlyrik will you still attempt to release this to a maven repo?

**Comment by computerlyrik** [comment]
tl;dr: No.

Long version:

@Xeli Since recently, i think it is the project maintainer who controls, which release, features etc. gets published. I may help @sovcik with release process if needed. Templates are available.

Personally, i am still investigating possible bugs in GPIO and Java-OSGI and trying to concentrate to solve these. As soon as possible i will step in with a helping hand.

**Comment by Xeli** [comment]
@computerlyrik that's a shame, but I understand it's better if the project maintainer has control over the maven repo.

I used your pom.xml's for compiling jpigpio though, perhaps you could add it to this repo for future use?

**Comment by nils-bauer** [comment]
Hi there!
Something new?

**Comment by Xeli** [comment]
I have published this library (or more specifically my fork) to the maven central repo: https://mvnrepository.com/artifact/eu.xeli/jpigpio_2.12

It does not contain the native api and uses the namespace eu.xeli.jpigpio but otherwise is identical to this library.

**Comment by mattjlewis** [comment]
FYI I've developed something similar that supports both local and remote socket access using pigpio, see https://github.com/mattjlewis/pigpioj
I am actively developing this as part of my diozero Java library at https://github.com/mattjlewis/diozero.

Matt

**Comment by nkolban** [comment]
Matt,
New users should most likely go with your project.  Great stuff.
